src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -42,30 +60,35 @@ internal Tensor(IntPtr handle)
             public override bool Equals(object? obj)
             {
                 return (obj is Tensor) && this.Equals((obj as Tensor)!);
-
             }
 
             /// <summary>
             ///  TBD
             /// </summary>
             /// <returns></returns>
-            public override int GetHashCode()
-            {
-                return base.GetHashCode();
-            }
+            public override int GetHashCode() => base.GetHashCode();
 
             /// <summary>
             /// A friendly name for the tensor. This is useful for debugging purposes.
             /// </summary>
             public string? name { get; set; }
 
+            /// <summary>
+            /// Helper method to determine if a tensor is disposed or not.
```

Comment:
Actually, I think calling it a 'Helper method' is underselling it. This is a bona fide state property of the tensor instance.

Reply:
Changed to

            /// <summary>
            /// Is true if the tensor has been disposed, false otherwise.
            /// </summary>

but the entire thing is a bit self explanatory.